backend/src/test/resources/application-default.yaml

```diff
@@ -228,3 +228,6 @@ simple-report-initialization:
       organization-external-id: DIS_ORG
     - patient-registration-link: inj3ct
       facility-name: Injection Site
+datahub:
+  url: "http://localhost:9561"
```

Comment:
So this port has a [binding issue when these tests run in CI](https://github.com/CDCgov/prime-simplereport/runs/6565681867?check_suite_focus=true).  Reading the docs and I set it to `0` which should result in a random port being used.  Issue now is that the mock responses seem to be firing off in the wrong order randomly, causing test failures.  🙃 

If anyone has any idea what I should be doing, I am all 👂 👂

Reply:
CI stack for context

```DatahubClientIntegrationTest > integrationTest_deserializesCorrectly() FAILED
    java.lang.IllegalStateException at DefaultCacheAwareContextLoaderDelegate.java:132
        Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException at AutowiredAnnotationBeanPostProcessor.java:643
            Caused by: org.springframework.beans.factory.BeanCreationException at AbstractAutowireCapableBeanFactory.java:1788
                Caused by: com.github.tomakehurst.wiremock.common.FatalStartupException at WireMockServer.java:149
                    Caused by: java.lang.RuntimeException at JettyHttpServer.java:198
                        Caused by: java.io.IOException at ServerConnector.java:349
                            Caused by: java.net.BindException at Net.java:-2
```

Reply:
solution: update configuration for wiremock to use `@AutoConfigureWireMock` and add `@DirtiesContext` to tests.